app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesActivity.java

```diff
@@ -16,6 +19,14 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         ImagesCompoundRecyclerView imagesCompoundRecyclerView =
                 (ImagesCompoundRecyclerView) findViewById(R.id.images_compound_recycler_view);
-        new ImagesViewModel(imagesCompoundRecyclerView, getContentResolver());
+        new ImagesViewModel(imagesCompoundRecyclerView,
```

Comment:
I prefer to move parameter under the Initializer like:
new ImagesViewModel(
                    imagesCompoundRecyclerView, 
                    etc,